### PR TITLE
Content for support for threadAllocatedBytes MXBean API corrected

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1199

Changed `getThreadAllocatedBytes MXBean` API to `com.sun.management.ThreadMXBean.getThreadAllocatedBytes()` API for consistency and clarity.

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/version0.41.md [hunk 1]
```diff
@@ -88,11 +88,11 @@ OpenSSL 3.x is now supported on all operating systems. For more information abou
 
 Performance of algorithms with the use of OpenSSL version 3 and later is now enhanced. Improved algorithms include SHA256, AES, HmacSHA256, and ChaCha20. To obtain these improved algorithms and further enhance the performance of these algorithms on Linux&reg; and AIX, use OpenSSL 3.0.12 or a later 3.0.x version, or 3.1.4 or a later 3.1.x version.
 
-### Support added for the `getThreadAllocatedBytes MXBean` API
+### Support added for the `com.sun.management.ThreadMXBean.getThreadAllocatedBytes()` API
 
-With this release, the OpenJ9 VM implementation supports thread memory allocation measurement (`getThreadAllocatedBytes com.sun.management.ThreadMXBean` API). The `isThreadAllocatedMemorySupported` method now returns true. The `setThreadAllocatedMemoryEnabled()` and `isThreadAllocatedMemoryEnabled()` methods are also supported now.
+With this release, the OpenJ9 VM implementation supports thread memory allocation measurement (`com.sun.management.ThreadMXBean.getThreadAllocatedBytes()` API). The `isThreadAllocatedMemorySupported()`, `setThreadAllocatedMemoryEnabled()`, and `isThreadAllocatedMemoryEnabled()` methods are supported now.
```
Comment (pshipton) [inline docs/version0.41.md:93]: See my comment on the release notes about the methods being "supported now".

File: docs/version0.41.md [hunk 2]
```diff
@@ -88,11 +88,11 @@ OpenSSL 3.x is now supported on all operating systems. For more information abou
...
 
-Support for the `getThreadAllocatedBytes MXBean` API is not added on z/OS&reg; platforms in this release.
+Support for the `com.sun.management.ThreadMXBean.getThreadAllocatedBytes()` API is not added on z/OS&reg; platforms in this release.
```
Comment (pshipton) [inline docs/version0.41.md:95]: Pls indicate it will be added in the next z/OS (24_01) release. We should add a release note to 0.43 to indicate this.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Pull request merged**